### PR TITLE
My stakeholder id percept

### PR DESCRIPTION
**Pull request by klikooo**
We can now percept `my_stakeholder_id(<ID>)`. I created a lot of new files so we can have several percepts per `Item`. With this we can make the stakeholder percept more clearer (instead of having 2 or 3 different percepts in it).

User story:
As a developer, I want the virtual human to be able to see who it is, so that it can query information about itself, its indicators and its belongings.

### CODE REVIEW

File: environment/src/main/java/tygronenv/EntityEventHandler.java [hunk 1]
```diff
@@ -89,7 +91,10 @@ private void notifyListener1(Event event) throws EntityException {
 		if (type instanceof MapLink) {
 			switch ((MapLink) type) {
 			case STAKEHOLDERS:
-				createPercepts(event.<ItemMap<Stakeholder>> getContent(MapLink.COMPLETE_COLLECTION), type);
+				ItemMap<Stakeholder> itemMap = event.<ItemMap<Stakeholder>> getContent(MapLink.COMPLETE_COLLECTION);
+				ArrayList<Stakeholder> items = new ArrayList<Stakeholder>(itemMap.values());
+				createExtraPercepts(items, type, new StakeholderPercepts(items, entity.getIntentedStakeholder()));
+//				createPercepts(event.<ItemMap<Stakeholder>> getContent(MapLink.COMPLETE_COLLECTION), type);
```
Comment (Denpeer) [inline environment/src/main/java/tygronenv/EntityEventHandler.java]: Why did you leave the old code?

File: environment/src/main/java/tygronenv/TygronEntity.java [hunk 2]
```diff
@@ -57,6 +57,7 @@
 	 */
 	public TygronEntity(EisEnv env, Stakeholder.Type stakeholdertype, Integer slotID) {
 		this.environment = env;
+		System.out.println("RICOOO: " + stakeholdertype);
```
Comment (MarcianoJ) [inline environment/src/main/java/tygronenv/TygronEntity.java]: You forgot to remove the debug code
EDIT: Nevermind, I see you deleted it in a later commit.

File: environment/src/main/java/tygronenv/TygronEntity.java [hunk 3]
```diff
@@ -70,6 +71,7 @@ public TygronEntity(EisEnv env, Stakeholder.Type stakeholdertype, Integer slotID
 	 */
 	public void notifyReady(String entity) throws EntityException {
 		stakeholder = getStakeholder(intendedStakeholder);
+		System.out.println("RICO notify" + intendedStakeholder);
```
Comment (MarcianoJ) [inline environment/src/main/java/tygronenv/TygronEntity.java]: Here also some debug code
EDIT: Nevermind, I see you deleted it in a later commit.

**Comment by MarcianoJ** [comment]
Approved!

**Comment by MaxGroenenboom** [comment]
Looks good, I approve.

**Comment by Denpeer** [comment]
Approve, let's merge

**Pull request merged**